game/BeepSaberMainMenu.gd

```diff
@@ -395,16 +388,6 @@ func _on_Exit_Button_pressed():
 	get_tree().quit()
 
 
-func _on_Restart_Button_pressed():
-	_beepsaber.restart_map();
-
-
-func _on_Continue_Button_pressed():
-	_beepsaber.continue_map();
-
-func _on_Stop_Button_pressed():
-	set_mode_game_start();
```

Comment:
These were a bunch of dangling signal callbacks that no longer get called. The mainmenu node no longer has the stop/start/continue buttons; instead, the EndSongPanel now has those buttons and the callback are made on the game main class itself.